File: C++/Problems/Problems/problem_3857.cpp
#include "problem_3857.h"

bool problem_3857::test() {
    int n = 3;

    int expected = 3;

    auto result = minCost(n);

    std::cout << result << std::endl;

    return result == expected;
}

int problem_3857::minCost(int n) {
    return n * (n - 1) / 2;
}

File: Python/Problems/Problems/problem3857.py
import heapq
import math
import sys

from utils import print1DArray, print2DArray
from typing import List, Set, Optional
from collections import Counter

from .problem import Problem


class Problem3857(Problem):
    def test(self) -> bool:
        n = 3

        expected = 3

        result = self.minCost(n)

        print(result)

        return result == expected

    def minCost(self, n: int) -> int:
        return n * (n - 1) // 2